### ISSUE TITLE
KITTI data to pcd

### ISSUE THREAD

**Issue by G12311231** [KTH-RPL/DynamicMap_Benchmark#14]
Hello,
thank you for your open source work!
And I would like to know how you generate the pcd file of the adaptation code from the raw kitti data.

**Comment by Kin-Zhang** [comment]
README file is here: https://github.com/KTH-RPL/DynamicMap_Benchmark/blob/master/scripts/README.md#open-source-dataset

script file is here: https://github.com/KTH-RPL/DynamicMap_Benchmark/blob/master/scripts/py/data/extract_semkitti.py

Let me know if you have any problem.

**Comment by G12311231** [comment]
Thank you for your response!
I encountered the following error while catkin_make the 'simple_ndt_slam' file:
Could not find a package configuration file provided by "catkin_simple"
  with any of the following names:

    catkin_simpleConfig.cmake
    catkin_simple-config.cmake
And I have found that the file is empty, how can I slove this problem.

In the meantime, I have a small request that can you please provide a download link for the KITTI01 data in PCD format for validation, if it is convenient for you?

**Comment by Kin-Zhang** [comment]
For SemanticKITTI, you don't need to run simple_ndt_slam, but follow the README I attached above (and also here again):

https://github.com/KTH-RPL/DynamicMap_Benchmark/blob/master/scripts/README.md#open-source-dataset

There is only a python script needs to run:

https://github.com/KTH-RPL/DynamicMap_Benchmark/blob/master/scripts/README.md#a1-extract-point-cloud-data

**Comment by G12311231** [comment]
Ok, I will try it now!
And I have a new question about how to identify dynamic points in the gt_cloud, I see that you mentioned label=1 in the gt_cloud as the indication for dynamic points you labeled. However, when I read the gt_cloud in my program, all the labels are equal to 0. Here is a snippet of my code:
pcl::io::loadPCDFile<pcl::PointXYZL>("/home/data/semindoor/gt_cloud.pcd", *raw);

    // 获取点云中的标签=1的数据（作者标记的动态点）
    for (const auto& point : *raw) {
        
        if(point.label == 1)
        {
            dynamic->push_back(point);
        }       
        std::cout << "点云标签: " << point.label << std::endl;
    }

**Comment by Kin-Zhang** [comment]
we save it in intensity metadata

pcl::io::loadPCDFilepcl::PointXYZI("/home/data/semindoor/gt_cloud.pcd", *raw);

**Comment by G12311231** [comment]
Thank you very much!

When running the modified ERASOR algorithm based on the semindoor data you provided, I noticed that the output does not remove dynamic points effectively. Is this behavior expected?
The command I used to run the algorithm is as follows:
ndc@ndc:~/ERASOR/build$ ./erasor_run /home/nrc/data/semindoor ../config/seq_semindoor.yaml

And it shows:
I0618 17:03:15.905598 2965445 ERASOR.cpp:70] Set Config for ERASOR
I0618 17:03:15.906394 2965445 ERASOR.cpp:84] ERASOR is initialized
W0618 17:03:15.906417 2965445 ERASOR_pcd.cpp:47] Raw map does not exist: /home/nrc/data/semindoor/raw_map.pcd, we will try to use the `gt_cloud` only for prior raw map.
I0618 17:03:16.846617 2965445 ERASOR_pcd.cpp:52] Raw map loaded, size: 26514322

ERASOR Timings:
	 Component		Total	Last	Mean	StDev	 Min	 Max	 Steps
	0. Read RawMap  	 3.29	3.2910	3.2910	  nan	3.2910	3.2910	     1
	4. Write        	 0.05	0.0506	0.0506	  nan	0.0506	0.0506	     1
I0618 17:03:20.190500 2965445 ERASOR_pcd.cpp:119] Done! Check the output in /home/nrc/data/semindoor/erasor_output.pcd
